.github/workflows/build-monorepo-runner.yml

```diff
@@ -0,0 +1,47 @@
+name: Test build in monorepo
+on:
+  pull_request:
+    paths:
+      - .github/workflows/build-monorepo.yml
+      - RNReanimated.podspec
+      - android/build.gradle
+  push:
+    branches:
+      - main
+
+jobs:
+  build_android_reanimated_in_app:
+    name: Build Android with in app Reanimated
+    uses: ./.github/workflows/build-monorepo.yml
+    with:
+      runs_on: ubuntu-latest
+      os: Android
+      concurrency_group: build_monorepo_android_in_app_reanimated
```

Comment:
We need to add `-${{ github.ref }}` here so the CI jobs are not cancelled for example when running for two separate PRs.